### PR TITLE
Tried to make lines fit below 100 characters long

### PR DESCRIPTION
**Pull request by mariari**
Also I cleaned up some trailing whitespace

This also fixes the code formatters issue with parsing `Typechcker`

### CODE REVIEW

File: src/Juvix/Core/IR/Typechecker.hs [hunk 1]
```diff
@@ -73,12 +73,17 @@ typeTerm ∷
   Term primTy primVal →
   Annotation primTy primVal m →
   m ()
-
--- * (Universe formation rule)
+-- ★ (Universe formation rule)
```
Comment (thealmarty) [inline src/Juvix/Core/IR/Typechecker.hs]: I noticed the error. Glad it's fixed!

File: src/Juvix/Core/IR/Typechecker.hs [hunk 2]
```diff
@@ -73,12 +73,17 @@ typeTerm ∷
...
 
 typeTerm _ _ii _g t@(Star i) ann = do
   typeTermIntroLog t ann
-  logOutput "patterned matched to be a * term. Type checker applies the universe formation rule. Checking that Sigma is zero. "
+  logOutput
```
Comment (thealmarty) [inline src/Juvix/Core/IR/Typechecker.hs]: This is fine but the spacing at the end were intentional. I think having a white space between sentences are easier to read.

Comment (mariari) [reply src/Juvix/Core/IR/Typechecker.hs]: Hmm, adding it here, seems rather error prone, as one can forget it. Could this be added to the logOutput function itself?

Comment (thealmarty) [reply src/Juvix/Core/IR/Typechecker.hs]: That may be a good idea. Can be tweaked further...I expect I'll have to change the logs along the way.

**Comment by mariari** [comment]
> Maybe revert the last commit? Doesn't matter much because I expect I'll have to edit the logging for readability anyway when I read the logs.

I'll probably just add them back, as it's shifted between a few commits.

I think the tricky thing here, is making it so the first log doesn't have an extra space, as if we add it to `logOutput` I suspect this will have to be programmed around

File: src/Juvix/Core/IR/Typechecker.hs (hunk 2 above)
Comment (thealmarty) [reply src/Juvix/Core/IR/Typechecker.hs]: I was going to take them out as a new datatype as Chris suggested. But after we've debugged more.

**Comment by thealmarty** [comment]
> > Maybe revert the last commit? Doesn't matter much because I expect I'll have to edit the logging for readability anyway when I read the logs.
> 
> I'll probably just add them back, as it's shifted between a few commits.
> 
> I think the tricky thing here, is making it so the first log doesn't have an extra space, as if we add it to `logOutput` I suspect this will have to be programmed around

Oh no I don't want you to spend so much time on it! We can merge as is. Again, I'll change it anyway.

**Pull request merged**